docs/ROADMAP.md

```diff
@@ -65,7 +65,11 @@ and Tekton Pipelines.
 
 ### Jupyter Requirements Management
 
-A tool focused on the Data Scientist workflow is [nb-requirements](https://github.com/thoth-station/jupyter-nbrequirements)
-and [jupyterlab-requirements](https://github.com/thoth-station/jupyterlab-requirements), it features a
-similar functionality as thamos, but embeds the dependencies and the locked dependencies within the meta information
-of the Jupyter Notebook file itself.
+Jupyter tools focused on the Data Scientist workflow have been created to help them in dependencies management:
```

Comment:
I think its "help with" not "help in" @harshad16 ?

Reply:
help with seems more suitable.